api/src/main/java/com/exelaration/abstractmemery/security/WebSecurity.java

```diff
@@ -37,7 +34,10 @@ protected void configure(HttpSecurity http) throws Exception {
         .csrf()
         .disable()
         .authorizeRequests()
-        .antMatchers(HttpMethod.POST, SIGN_UP_URL)
+        // .antMatchers(HttpMethod.POST, SIGN_UP_URL)
+        .antMatchers(
+            "/**") // this line should be removed and the line above uncommented once we get jwts
```

Comment:
Since we now know the functionality works, would we want to revert this back for the time being?

Reply:
Can do! Are those changes in master now?

Reply:
the JWTs, no. I meant since we know the direct links functionality works with this pr

Reply:
Will do